app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+# Controller for Groups
+class GroupsController < ApplicationController
+  prepend_before_action :set_group, only: %i(show edit update destroy)
+  before_action :set_draw
+
+  def show; end
+
+  def new
+    @group = Group.new(draw: @draw)
+  end
+
+  def create
+    p = group_params.to_h
+    p[:leader_id] = current_user.id unless current_user.admin?
+    result = GroupCreator.new(p).create!
```

Comment:
I know this is probably a minor point, but aren't we passing in a `Hash` here instead of an `ActionController::Parameters` object as we've done in the other controllers / creators (and as we document in the `GroupCreator`)? With duck typing it shouldn't matter, but we should update the documentation in the `GroupCreator` to be accurate.